### DISCUSSION TITLE
Realized Wiki did not have change approvals enabled.

### DISCUSSION THREAD

**Discussion by ProjectInitiative** [seaweedfs/seaweedfs#2265] [General]
Hello! 
I edited [this](https://github.com/chrislusf/seaweedfs/wiki/Failover-Master-Server) wiki page thinking that it would be submitted for approval, and not just implemented as a live change. If someone could please review and roll back if the change I made was incorrect please let me know! 

Thanks!

**Comment by chrislusf** [comment]
Looks good. Thanks!
